src/main.rs

```diff
@@ -12,14 +14,17 @@ use arguments::Command::Running;
 use arguments::Command::Start;
 use arguments::Command::Stop;
 use arguments::CommandLineArguments;
-use chrono::Utc;
 use colored::Colorize;
-use credentials::Credentials;
-use models::{ResultWithDefaultError, TimeEntry};
+use commands::auth::AuthenticationCommand;
+use commands::cont::ContinueCommand;
```

Comment:
`cont` sounds a little risque, could we use the full name `continue` like the `running` module?

Reply:
I tried, but continue is a reserved word so vscode autocompletion looked weird 😭 

But I guess we can live with that if the compiler doesn't complain

Reply:
ahh, alrighty 👍